core-common/src/main/java/org/mapstruct/Context.java

```diff
@@ -25,14 +25,22 @@
 
 /**
  * Marks a parameter of a method to be treated as <em>mapping context</em>. Such parameters are passed to other mapping
- * methods, {@link ObjectFactory} methods or {@link BeforeMapping}/{@link AfterMapping} methods when applicable and can
- * thus be used in custom code. The {@link Context} parameters are otherwise ignored by MapStruct.
+ * methods, {@code @}{@link ObjectFactory} methods or {@code @}{@link BeforeMapping}/{@code @}{@link AfterMapping}
+ * methods when applicable and can thus be used in custom code.
  * <p>
- * For generated code to call a method that is declared with {@link Context} parameters, the declaration of the mapping
- * method being generated needs to contain at least those (or assignable) {@link Context} parameters as well. MapStruct
- * will not create new instances of missing {@link Context} parameters nor will it pass {@code null} instead.
+ * The type of an {@code @Context} parameter is also inspected for factory methods (with or without the {@code @}
```

Comment:
Nothing special, but why not use `{@code @}{@link Context}` like the other annotations?